src/client/app/containers/BarChartContainer.js

```diff
@@ -84,7 +84,7 @@ function mapStateToProps(state) {
 				stacked: state.graph.barStacking,
 				scaleLabel: {
 					display: true,
-					labelString: 'kWh'
+					labelString: 'kW'
```

Comment:
The bar chart is still kWh

Reply:
What about the compare chart?

Reply:
I'm not sure. My guess is it's kWh, but it depends on where it's getting its data. It probably should be kWh.

Reply:
Issue #204 says that all should be KW. The rationale is that you will see the same units as you change representations. Also, for compare, it will not cause the bars to change height so it is only a units question. Also, since the bar graph is related to the compare I think it is best if they are the same.

Reply:
I thought we were displaying a different your of data for the bar chart. I just looked at the SQL and it should all be kW. I was wrong.